src/docs/resources/faq.md

```diff
@@ -208,7 +208,7 @@ work.
 
 Debug mode builds use a virtual machine (VM) to run Dart code (hence the
 "debug" banner they show to remind people that they're slightly slower)
-in order to enable stateful hot reload.
+in order to enable Stateful Hot Reload.
```

Comment:
Upper case is fine, but I could go either way here.

Basically, when using "English", it's ok to use lower case. So, you would write "enable the widget...", not "enable the Widget".  But if you are, say, writing about a specific class or instance or method, then you would code font and use proper case.

Reply:
Cool, will leave this as upper case here.